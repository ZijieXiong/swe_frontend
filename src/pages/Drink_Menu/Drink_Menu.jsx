import React from 'react'
import {useEffect} from 'react'
//Utilizing foodmenu_css 


export default function Drink_Menu({drinkName}) {
    
    return (
        <div>
            <p>{drinkName}</p>
        </div>
    )
}
