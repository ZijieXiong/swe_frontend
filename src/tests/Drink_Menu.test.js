import React from 'react'
import renderer from 'react-test-renderer'
import TestRenderer from 'react-test-renderer';

import { MemoryRouter } from 'react-router-dom';
import { render, screen } from '@testing-library/react';
//npm install --save-dev react-test-renderer

import Drink_Menu from '../pages/Drink_Menu/Drink_Menu.jsx'
it('renders correctly', () => {
  const tree = renderer
    .create(<Drink_Menu/>)
    .toJSON();
  expect(tree).toMatchSnapshot();
});

