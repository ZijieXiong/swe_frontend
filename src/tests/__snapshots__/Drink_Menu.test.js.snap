// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="food_menu"
>
  <div
    className="extra_space"
  />
</div>
`;
